src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -179,7 +177,7 @@ private function createTag($name)
         return $tag;
     }
 
-    private function createMedia($title, $collection, $mimeType, $type, $tags = [], $targetGroups = [])
+    private function createMedia($title, $collection, $mimeType, $type, $tags = [], $targetGroups = []): Media
```

Comment:
we should reallly focus just on the PHPUnit update this get lot of hard to review things if we do here such refactorings.

Reply:
test method changes should be done on 2.5 branch if possible and non breaking changes